packages/ember-application/lib/system/application.js

```diff
@@ -331,6 +331,13 @@ var Application = Namespace.extend(DeferredMixin, {
   buildDefaultInstance: function() {
     var instance = this.buildInstance();
 
+    // For the default instance only, set the view registry to the global
+    // Ember.View.views hash for backwards-compatibility.
+    var registry = instance.applicationRegistry;
```

Comment:
wouldn't it be simpler to do 
`EmberView.views = instance.container.lookup('-view-registry:main')`